doc/modules/pipeline.rst

```diff
@@ -37,8 +37,8 @@ is an estimator object::
     >>> from sklearn.pipeline import Pipeline
     >>> from sklearn.svm import SVC
     >>> from sklearn.decomposition import PCA
-    >>> estimators = [('reduce_dim', PCA()), ('clf', SVC())]
-    >>> pipe = Pipeline(estimators)
+    >>> param_grid = [('reduce_dim', PCA()), ('clf', SVC())]
+    >>> pipe = Pipeline(param_grid)
```

Comment:
Why param_grid? This is not a parameter grid, but a list of names and estimators. I am -1 on this change.